ckan/cli/db.py

```diff
@@ -0,0 +1,76 @@
+# encoding: utf-8
+
+import os
+
+import click
+from flask import Flask, current_app
+from werkzeug.serving import run_simple
+
+from ckan.cli import click_config_option
+import logging
+
+
+log = logging.getLogger(__name__)
+
+
+@click.group(name=u'db', short_help=u'Database commands')
+@click.help_option(u'-h', u'--help')
```

Comment:
I assume this is to support `-h` in additon to `--help`?  It seems overkill to have to define this for every single command and subcommand. IIUC, [this](http://click.palletsprojects.com/en/6.x/documentation/#help-parameter-customization) should set it globally at the `ckan` command level, so we can remove all the individual decorators